index.html

```diff
@@ -195,97 +208,108 @@ <h2 class="project-name">Project name goes here</h2>
                             </button>
                         </div>
                     </div>
-                </div>
-        </section>
-        <section class="about">
-            <div>
-                <h2 class="headings">About</h2>
-                <div class="content">
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                <button class="button download">Get my CV
-                    <div class="icon"></div>
-                </button>
-                </div>
-            </div>
-            <section class="competency">
+           </section> 
+ </section>
+  <!-- end of project section -->
+
+   <!-- begining of about section -->
+   <section class="about">
+    <div class="about-text">
+        <h2 class="headings">About</h2>
+        <div class="content desk">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button class="button download">Get my CV
+            <div class="icon"></div>
+        </button>
+        </div>
+    </div>
+    <section class="competency">
+        <ul class="grid">
+            <li class="language grid-item">
+                <h3>Languages</h3>
```

Comment:
- kindly clear unnecessary spaces above and below the about section to stick to the given design and have a better Ui 👍
### Figma design 👇
![Screenshot 2023-05-12 214325](https://github.com/ctokoli/personal-portfolio-site/assets/52010789/df4e91ed-b11d-4105-929d-95e8db28c33a)

### Your Design 👇
![Screenshot 2023-05-12 214524](https://github.com/ctokoli/personal-portfolio-site/assets/52010789/dad59fbe-6aa8-48be-9944-afef144de555)
![Screenshot 2023-05-12 214447](https://github.com/ctokoli/personal-portfolio-site/assets/52010789/4f8f57b7-44da-42cf-b5cb-cd95a51e0196)